### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by mayorova**
Nginx tests

### CODE REVIEW

File: t/blackbox.t [hunk 1]
```diff
@@ -75,7 +74,6 @@ success!
 --- error_code: 200
 
 
-
```
Comment (mikz) [inline t/blackbox.t]: This whitespace was intentional :) openresty projects enforce this to be 3 spaces, so would be good to follow the lead.

Comment (mayorova) [reply t/blackbox.t]: oh! so, 3 spaces between each test?

Comment (mikz) [reply t/blackbox.t]: Yep. Hopefully we will enforce it by some linter at some point.

File: README.md [hunk 2]
```diff
@@ -18,8 +18,13 @@ Build process uses s2i to package docker image.
 s2i build . quay.io/3scale/s2i-openresty-centos7:1.11.2.5-1-rover2  cors-proxy-app
 ```
 
+Run the container:
+```shell
+docker run --rm --name corsproxy -p 8080:8080 cors-proxy-app
```
Comment (mikz) [inline README.md:23]: This should be updated in https://github.com/3scale/cors-proxy/pull/32 IMO.

**Review by mikz — APPROVED** [review]
🥇 great! much appreciated!

File: README.md (hunk 2 above)
Comment (mayorova) [reply README.md]: I can rebase #32 after this one is merged.

**Pull request merged**